rebase.go

```diff
@@ -78,7 +82,7 @@ func (c *Client) Rebase(ctx context.Context, opts RebaseOptions) error {
 	}
 
 	c.logger.Infof("Rebasing %s on run image %s", style.Symbol(appImage.Name()), style.Symbol(baseImage.Name()))
-	rebaser := &lifecycle.Rebaser{Logger: c.logger}
+	rebaser := &lifecycle.Rebaser{Logger: c.logger, PlatformAPI: api.Platform.Latest()}
```

Comment:
I don't really know what the correct move is here.

My rationale is that one usually get the platform API version from the builder, but there's no builder in this scenario.

Reply:
My initial inclination was that it might have to do with the image being rebased but that was proven wrong when I looked at the [implementation](https://sourcegraph.com/github.com/buildpacks/lifecycle@a7428a55c2a14d8a37e84285b95dc63192e3264e/-/blob/rebaser.go#L91-94). It seems to only affect the output of the report.

That said, I don't think this should be `Latest`. I think **it should be hardcoded to a version that pack supports**. My fear is that this gets bumped when updating lifecycle version and _something_ unexpected changes. We should be conscious of all updates.